### ISSUE TITLE
[Feature Request] View Grid

### ISSUE THREAD

**Issue by Wilson159** [SebLague/Digital-Logic-Sim#154]
I like to be very neat when playing around with this simulator, it would be cool if there was a `Grid View` that also has an option `Snap to Grid`!

Thanks!

**Comment by SebLague** [comment]
Agreed! I'm currently trying to figure out how the snapping should behave with chips that have differing input/output pin counts (see #132 )

**Comment by SebLague** [comment]
Added in new version
